ui/src/components/PollBody/PollBody.jsx

```diff
@@ -0,0 +1,338 @@
+import React, { useState, useEffect } from 'react';
+import AutoSuggest from 'react-autosuggest';
+import Voter from '../Voter/Voter';
+import AnswerOption from '../AnswerOption/AnswerOption';
+import API from '../../API';
+import members from '../../data/voters.json';
+import './PollBody.css';
+
+const PollBody = () => {
+    const [currPage, setCurrPage] = useState(0);
+    const [privacy, setPrivacy] = useState('private');
+    const [value, setValue] = useState('');
+    const [suggestions, setSuggestions] = useState([]);
+
+    const [pollTitle, setPollTitle] = useState("");
+    const [pollDescription, setPollDescription] = useState("");
+    const [pollType, setPollType] = useState("");
+    const [pollExpiration, setPollExpiration] = useState();
+    const [voters, setVoters] = useState([]);
+    const [optionInds, setOptionInds] = useState([1, 2]);
+    const [numForceUpdates, setNumForceUpdates] = useState(0); // BAD PRACTICE ;-;
```

Comment:
Yes, this is probably bad practice :)

Reply:
Actually, I have tried useEffect before throwing in another hook, and useEffect didn't actually fire on setOptionInds - according to google searches, it's  probably because the array itself didn't change.

I will try putting the pages into their own components, and see if that works :D

Reply:
Just realized `useState()` only updates if the underlying variable pointer changes, not if internal values of the object change. Only `useReducer()` manages that properly.

I can see you fixed this in the latest commit, so that's great!